pkg/agent/cniserver/pod_configuration_windows.go

```diff
@@ -134,3 +135,169 @@ func (pc *podConfigurator) reconcileMissingPods(ifConfigs []*interfacestore.Inte
 		}
 	}
 }
+
+type unReadyPodInfo struct {
+	podName      string
+	podNamespace string
+	annotated    bool
+	createTime   time.Time
+}
+
+type podIfaceMonitor struct {
```

Comment:
when I look at all this new code, it seems that there is nothing platform-specific about it: it could build and run on Linux just as well as on Windows (unless I am missing something). Given this, could we move the `podInfaceMonitor` code to a separate file or sub-package, that will not be platform-specific. It will make it much easier to update that code and unit test it locally on macOS laptops or Linux dev servers. I think this code deserved to be unit-tested independently of the rest of the pod configuration code.

On Windows, we can use a pointer to an actual instance of `podIfaceMonitor`, and on Linux, we can use a nil pointer.